chart/values.yaml

```diff
@@ -0,0 +1,3071 @@
+    ## @param workspaceWww.serviceAccount.create Specifies whether a ServiceAccount should be created
+    ##
+    create: true
+    ## @param workspaceWww.serviceAccount.name The name of the ServiceAccount to use.
+    ## If not set and create is true, a name is generated using the common.names.fullname template
+    ##
+    name: ""
+    ## @param workspaceWww.serviceAccount.automountServiceAccountToken Mount service account token in the deployment
+    ##
+    automountServiceAccountToken: false
+
+## @section Init Container Parameters
+##
+workspaceMigrations:
+  ## Migrations image
+  ## @param workspaceMigrations.image.registry workspace-db image registry
+  ## @param workspaceMigrations.image.repository workspace-db image repository
+  ## @param workspaceMigrations.image.tag workspace-db image tag (immutable tags are recommended)
+  ## @param workspaceMigrations.image.pullPolicy workspace-db image pull policy
+  ## @param workspaceMigrations.image.pullSecrets workspace-db image pull secrets
+  ##
+  image:
+    registry: gcr.io
+    repository: carto-onprem-artifacts/workspace-db
+    tag: "2021.12.20"
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ## e.g:
+    ## pullSecrets:
+    ##   - myRegistryKeySecretName
+    ##
+    pullSecrets: []
+  ## @param workspaceMigrations.command Override default container command (useful when using custom images)
+  ##
+  command: []
+  ## @param workspaceMigrations.args Override default container args (useful when using custom images)
+  ##
+  args: []
+  ## Init container's resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## @param workspaceMigrations.resources.limits The resources limits for the init container
+  ## @param workspaceMigrations.resources.requests The requested resources for the init container
+  ##
+  resources:
+    limits: {}
+    requests: {}
+  ## Init container Container Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+  ## @param workspaceMigrations.containerSecurityContext.enabled Enable container security context
+  ## @param workspaceMigrations.containerSecurityContext.runAsUser Set init container's Security Context runAsUser
+  ## @param workspaceMigrations.containerSecurityContext.runAsNonRoot Force the init container to run as non root
+  ## NOTE: when runAsUser is set to special value "auto", init container will try to chown the
+  ##   data folder to auto-determined user&group, using commands: `id -u`:`id -G | cut -d" " -f2`
+  ##   "auto" is especially useful for OpenShift which has scc with dynamic user ids (and 0 is not allowed)
+  ##
+  containerSecurityContext:
+    enabled: true
+    runAsUser: 0
+    runAsNonRoot: false
+
```

Comment:
Some generic comments:

- Do we require any RBAC rule?
- NetworkPolicies?
- PDB(s) or AutoScaling?
- Metrics? If so, ServiceMonitor for Prometheus Operator?

Reply:
RBAC:
  - It seems that not at the moment.
Netpol, HPA or PDB
  - We can check with the CARTO team, I'd save it for the meeting on the 30th but I don't think this is necessary for the first cut.

Metrics:
  -  I've checked their code and they don't seem to have metrics endpoints. I'll ask them in the meeting.